indexer/migrations/20230523_create_schema.sql

```diff
@@ -80,10 +86,42 @@ CREATE TABLE IF NOT EXISTS output_proposals (
     l1_contract_event_guid VARCHAR REFERENCES l1_contract_events(guid)
 );
 
+/**
+ * Derived data
+ * These schemas may change and need to resync
+ * We should consider syncing this from the CHAIN data instead of directly
+ * from the processor
+ */
+
 /**
  * BRIDGING DATA
  */
 
+ /**
+ * TOKEN DATA
+ */
+
+-- L1 Token table
+CREATE TABLE IF NOT EXISTS l1_tokens (
+    token_address   VARCHAR PRIMARY KEY,
+    name            VARCHAR NOT NULL,
+    symbol          VARCHAR NOT NULL,
+    decimals        INTEGER NOT NULL CHECK (decimals >= 0 AND decimals <= 18),
+    bridge_address  VARCHAR NOT NULL,
+    l2_token_address VARCHAR NOT NULL
+);
+
+-- L2 Token table
+CREATE TABLE IF NOT EXISTS l2_tokens (
+    token_address   VARCHAR PRIMARY KEY,
+    name            VARCHAR NOT NULL,
+    symbol          VARCHAR NOT NULL,
+    decimals        INTEGER NOT NULL CHECK (decimals >= 0 AND decimals <= 18),
+    l1_token_address VARCHAR REFERENCES l1_tokens(token_address),
+    bridge_address  VARCHAR NOT NULL
```

Comment:
super nit: same order of fields as `l1_tokens`. bridge_address before `l1_token_address`